src/snapshot/prepare_snapshot_received_proposals_airtable.js

```diff
@@ -51,45 +51,16 @@ const prepareProposalsForSnapshot = async (curRound) => {
 
   await Promise.all(
     proposals.map(async (proposal) => {
-      getProposalRecord(proposal, proposals)
-      try {
-        const wallet_0x = proposal.get('Wallet Address')
-        const proposalStanding = proposal.get('Proposal Standing')
-
-        // Please update enums as required
-        const goodStanding =
-          proposalStanding === Standings.Completed ||
-          proposalStanding === Standings.Refunded ||
-          proposalStanding === Standings.Undefined ||
-          proposalStanding === Standings.Unreported ||
-          proposalStanding === Standings.NewProject
-
-        if (hasEnoughOceans(wallet_0x) && goodStanding === true) {
-          recordsPayload.push({
-            id: proposal.id,
-            fields: {
-              'Proposal State': 'Accepted',
-              'Voting Starts': voteStartTime,
-              'Voting Ends': voteEndTime,
-              'Snapshot Block': Number(estimatedBlockHeight),
-              'Deployment Ready': 'Yes'
-            }
-          })
-        } else {
-          recordsPayload.push({
-            id: proposal.id,
-            fields: {
-              'Proposal State': 'Rejected',
-              'Voting Starts': null,
-              'Voting Ends': null,
-              'Snapshot Block': null,
-              'Deployment Ready': 'No'
-            }
-          })
+      recordsPayload.push({
+        id: proposal.id,
```

Comment:
We don't need to check the 500 Oceans required here because Daobot checks that every minute and will set the proposal to Rejected state if the wallet doesn't have enough Ocean tokens.